### ISSUE TITLE
Can't load sf2 in neothesia-cli

### ISSUE THREAD

**Issue by kawayuta** [PolyMeilex/Neothesia#7]
Can't load sf2 in neothesia-cli
. /out/video.mp4
How do I include sound in exported video?

**Comment by MonliH** [comment]
Yes, I was also wondering if there was a way to export audio using the cli application.
Currently, recording the screen and desktop audio seems like a very sub-optimal way to do it (and it's quite slow)

**Comment by PolyMeilex** [comment]
The main reason why I never worked on that is that I usually get my audio from a proper VTS. I believe that sound fonts are just not good enough, but I'm open to implementing this (or accepting a PR) in the future, so the that's why the issue stays open as a reminder.

Thanks for your interest!

**Comment by PolyMeilex** [comment]
I mostly need help with encoding the audio via ffmpeg, all the ffmped code is enclosed in the [mpeg_encoder](https://github.com/PolyMeilex/Neothesia/tree/master/mpeg_encoder) crate. Once there is a method for encoding video with audio, I can probably handle the rest myself by plugging my synth code into it.

This ffmpeg example probably explains decently how to encode audio, but I never had time to read it to figure out what kind of data format it excepts: http://ffmpeg.org/doxygen/trunk/muxing_8c-source.html
